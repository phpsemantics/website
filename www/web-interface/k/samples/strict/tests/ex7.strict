1+++(*p=++*(q=&y)+++++x)
