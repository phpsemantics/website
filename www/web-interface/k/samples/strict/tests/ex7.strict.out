<k>
    35
</k>
<store>
    lx |-> 34
    ly |-> 21
    lz |-> 30
    lp |-> lx
    lq |-> ly
</store>
<env>
    x |-> lx
    y |-> ly
    z |-> lz
    p |-> lp
    q |-> lq
</env>
